---
int8 value 
# Return values:
#  0..100   := current volume